portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +27,122 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  
+  /**
+   *This welcome String will remain here for now as it is still 
+   *in one of my PRs and I am unsure as to what the best practice 
+   *is for these cases. In the meantime, this String will be sent as
+   *part of the JSON response.
+  */
```

Comment:
I was looking at the commits for this PR and it's built on the commits for your other PRs. So as long as you merge the PRs in order (i.e. by oldest commit/branch), you can delete the code here and you won't have any merge conflicts. Also feel free to also create comments in your PR for this sorts of purposes, since you might not always want this comment accidentally committed in the code